wandb/apis/public.py

```diff
@@ -2439,7 +2436,11 @@ class Artifact(object):
     )
 
     @classmethod
-    def from_id(cls, client, id):
+    def from_id(cls, artifact_id, client, use_cache=True):
```

Comment:
Does use_cache really need to be optional?

Reply:
No, does not need to be optional. My intention here would be to have using the cache be default behavior and the programmer would need to make the explicit decision to override this behavior. If you prefer not to use the default, happy to change - not wed to the decision.

Reply:
I think we should only make it optional if we have some caller code that needs the option

Reply:
got it - updated in incoming commit